### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,22 @@
-Mercator 2.0
+mercator-go
 ============
 
-Mercator is a Swiss army knife for obtaining package metadata across various package ecosystems:
+`mercator` is a tool that can find packages on the filesystem and it can extract metadata about those packages.
+
+## Installation
+
+The easiest way how to install `mercator` is to install it from RPM package
```
Comment (tisnik) [inline README.md]: easiest/preferred?

**Comment by tisnik** [comment]
Error: Login invalid/expired. Please visit https://copr.fedorainfracloud.org/api to get or renew your API token.

File: README.md (hunk 1 above)
Comment (msrb) [reply README.md]: Easiest :)

**Comment by msrb** [comment]
[test]

**Comment by msrb** [comment]
[test]

**Comment by msrb** [comment]
[test]

**Pull request merged**